source/common/http/conn_manager_impl.cc

```diff
@@ -496,6 +496,7 @@ void ConnectionManagerImpl::RdsRouteConfigUpdateRequester::requestRouteConfigUpd
   } else if (parent_.snapped_scoped_routes_config_ != nullptr) {
     Router::ScopeKeyPtr scope_key = parent_.snapped_scoped_routes_config_->computeScopeKey(
         *parent_.filter_manager_.requestHeaders());
+    // If scope_key is not null, the scope exists but RouteConfiguration is not initialized.
     if (scope_key != nullptr) {
       // On demand srds
```

Comment:
nit: the func name is pretty straightforward, consider removing this line.

Reply:
I have removed it.